staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -99,10 +112,18 @@ for GVs in ${GROUPS_WITH_VERSIONS}; do
   for V in ${Vs//,/ }; do
     ALL_FQ_APIS+=("${EXT_APIS_PKG}/${G}/${V}")
     EXT_FQ_APIS+=("${EXT_APIS_PKG}/${G}/${V}")
+    GROUP_VERSIONS+=("${G}/${V}")
   done
 done
 
 if grep -qw "deepcopy" <<<"${GENS}"; then
+  # Nuke existing files
+  for dir in $(GO111MODULE=on go list -f '{{.Dir}}' "${ALL_FQ_APIS[@]}"); do
+    pushd "${dir}" >/dev/null
```

Comment:
do we also need to check for existence before doing pushd? if it doesn't exist, do we stay in the same directory and then find/remove all files under the original dir? (comment applies to all `push ... find/remove ... popd` instances)

Reply:
`pushd` will fail and trigger errexit.  `go list` operates on what IS there, rather than what should be there (unlike git ls-files).  So assuming that `go list` produced an output, that output should exist (unless we lost a race against someone else removing it at the same instant), and so pushd failing and aborting seems OK.

Really, if we want the best possible results, we would check the errorcode of EVERY COMMAND, but who wants to do that? :)

Reply:
> `pushd` will fail and trigger errexit. `go list` operates on what IS there, rather than what should be there

fail fast sgtm, I just wanted to make sure a failure wasn't going to nuke other stuff